### PR TITLE
[FLINK-23716][docs-zh] Translate the 'Zipping Elements in a DataSet' page inito Chinese

### PR DESCRIPTION
**Pull request by RocMarshal**
## What is the purpose of the change

*Translate the 'Zipping Elements in a DataSet' page inito Chinese*

## Brief change log

*Translate the 'Zipping Elements in a DataSet' page inito Chinese*

## Verifying this change

*Translate the 'Zipping Elements in a DataSet' page inito Chinese*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (**not applicable** / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by RocMarshal** [comment]
Hi, @wuchong Could you help me to review this pr ? Thank you.

File: docs/content.zh/docs/dev/dataset/zip_elements_guide.md [hunk 1]
```diff
@@ -25,17 +25,22 @@ specific language governing permissions and limitations
...
 
-`zipWithIndex` assigns consecutive labels to the elements, receiving a data set as input and returning a new data set of `(unique id, initial value)` 2-tuples.
-This process requires two passes, first counting then labeling elements, and cannot be pipelined due to the synchronization of counts.
-The alternative `zipWithUniqueId` works in a pipelined fashion and is preferred when a unique labeling is sufficient.
-For example, the following code:
+<a name="zip-with-a-dense-index"></a>
+
+### 以密集索引编号
+
+`zipWithIndex` 为元素分配连续的标签，接收数据集作为输入并返回一个新的 `(unique id, initial value)` 2-tuples 的数据集。
```
Comment (wuchong) [inline docs/content.zh/docs/dev/dataset/zip_elements_guide.md:39]:
```suggestion
`zipWithIndex` 为元素分配连续的标签，接收数据集作为输入并返回一个新的 `(unique id, initial value)` 二元组的数据集。
```

File: docs/content.zh/docs/dev/dataset/zip_elements_guide.md [hunk 2]
```diff
@@ -80,15 +85,17 @@ env.execute()
...
-### Zip with a Unique Identifier
+<a name="zip-with-a-unique-identifier"></a>
+
+### 以唯一标识符编号
 
-In many cases one may not need to assign consecutive labels.
-`zipWithUniqueId` works in a pipelined fashion, speeding up the label assignment process. This method receives a data set as input and returns a new data set of `(unique id, initial value)` 2-tuples.
-For example, the following code:
+在许多情况下，可能不需要指定连续的标签。
+`zipWithUniqueId` 以 pipelined 的方式工作，加快了标签分配的过程。该方法接收一个数据集作为输入，并返回一个新的 `(unique id, initial value)` 2-tuples 的数据集。
```
Comment (wuchong) [inline docs/content.zh/docs/dev/dataset/zip_elements_guide.md:97]:
```suggestion
`zipWithUniqueId` 以 pipelined 的方式工作，加快了标签分配的过程。该方法接收一个数据集作为输入，并返回一个新的 `(unique id, initial value)` 二元组的数据集。
```

**Comment by RocMarshal** [comment]
@wuchong  Updated based on your suggestions. Thank you very much.

**Review by wuchong — APPROVED** [review]
LGTM.

**Pull request merged**